catalog/rest/catalog-rest-endpoint/pom.xml

```diff
@@ -137,22 +245,48 @@
                     <instructions>
                         <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
                         <Embed-Dependency>
-                            json-smart,
-                            catalog-core-api-impl,
+                            asm,
                             catalog-core-actions,
+                            catalog-core-api-impl,
                             common-system,
-                            platform-util,
                             commons-lang3,
-                            asm
+                            json-smart,
+                            platform-util
                         </Embed-Dependency>
                         <Import-Package>
+                            !org.codice.ddf.platform.util,
                             javax.ws.rs.*,
                             *
                         </Import-Package>
                     </instructions>
                 </configuration>
             </plugin>
             <plugin>
+                <artifactId>maven-failsafe-plugin</artifactId>
+                <!-- Need to use an older version of failsafe for versionAsInProject() to work -->
+                <version>2.17</version>
```

Comment:
❓ should we use a property for the failsafe version so we can use the same version throughout?

Reply:
Looks like we're using different versions of failsafe throughout the project so we would need to make sure everything still works as expected if we wanted to make that change.